doc/api/perf_hooks.md

```diff
@@ -432,70 +432,104 @@ console.log(h.percentile(50));
 console.log(h.percentile(99));
 ```
 
-### Class: Histogram
+## perf_hooks.monitorRequestWrapLatency()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Histogram}
+
+Creates a `Histogram` object that reports the latency of livuv
+request handles. A request handle is an async resource within
+Node.js that tracks the state of an individual asynchronous
+operation (e.g. a file read, a DNS lookup, etc). The latency
+of a request is the length of time from when the request is
+passed off to the event loop to when the corresponding callback
+is invoked.
+
+The histogram effectively measures how long the event loop is
+taking to perform and resolve individual asynhronous operations,
+without differentiating between the types of operations
+measured.
+
+```js
+const [ monitorReqWrapLatency ] = request('perf_hooks');
+const h = monitorReqWrapLatency();
+h.enable();
+// Do something.
+h.disable();
+console.log(h.min);
+console.log(h.max);
+console.log(h.mean);
+console.log(h.stddev);
+console.log(h.percentiles);
+console.log(h.percentile(50));
+console.log(h.percentile(99));
+```
+
+## Class: Histogram
 <!-- YAML
 added: v11.10.0
 -->
-Tracks the event loop delay at a given sampling rate.
+Tracks sampled data values.
 
-#### histogram.disable()
+### histogram.disable()
 <!-- YAML
 added: v11.10.0
 -->
 
 * Returns: {boolean}
 
-Disables the event loop delay sample timer. Returns `true` if the timer was
-stopped, `false` if it was already stopped.
+Disables the histogram tracking. Returns `true` if the tracking was
+disabled, `false` if it was already disabled.
 
-#### histogram.enable()
+### histogram.enable()
 <!-- YAML
 added: v11.10.0
 -->
 
 * Returns: {boolean}
 
-Enables the event loop delay sample timer. Returns `true` if the timer was
-started, `false` if it was already started.
+Enables the histogram tracking. Returns `true` if the tracking was
+enabled, `false` if it was already enabled.
 
-#### histogram.exceeds
+### histogram.exceeds
 <!-- YAML
 added: v11.10.0
 -->
 
 * {number}
 
-The number of times the event loop delay exceeded the maximum 1 hour event
```

Comment:
Should the 1-hour maximum threshold for event loop delay be documented in the other place?

Reply:
Yeah, good point